recipes/plasmidid/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin/config_files
+mkdir -p $PREFIX/config_files
+
+cp bin/* $PREFIX/bin
+cp plasmidID $PREFIX/bin
+
+cp config_files/* $PREFIX/bin/config_files
+
+echo "INSTALING MASH 2"
+echo "#############################################"
+
+wget https://github.com/marbl/Mash/releases/download/v2.2/mash-Linux64-v2.2.tar
+tar -xvf mash-Linux64-v2.2.tar
+cp mash-Linux64-v2.2/* $PREFIX/bin
+
+echo "INSTALING CIRCOS WITH CUSTOM SETTINGS"
+echo "#############################################"
+
+wget http://circos.ca/distribution/circos-0.69-8.tgz
```

Comment:
This is the same problem as with Mash: Have you tried just adding a `circos` dependency? There exists a package for it. If the custom settings are the only reason for doing it like this, there must be a different way.

If you include other packages in your recipe like this, you get all sorts of problems. For example, if someone installs both plasmidid and circos into the same environment, the files conflict with each other. Perhaps it could even happen that someone explicitly installs a circos version that is not 0.69, but then gets the version from plasmidid.